zebra-chain/src/fmt.rs

```diff
@@ -2,28 +2,68 @@
 
 use std::{fmt, ops};
 
+#[cfg(any(test, feature = "proptest-impl"))]
+use proptest::prelude::*;
+#[cfg(any(test, feature = "proptest-impl"))]
+use proptest_derive::Arbitrary;
+
+/// Wrapper to override `Debug`, redirecting it to the type's name.
```

Comment:
Suggestion: make it a bit clearer.

```suggestion
/// Wrapper to override `Debug`, redirecting it to only output the type's name.
```